server/reports/attendance/attendance-endpoint.js

```diff
@@ -71,7 +71,6 @@ export function getAttendanceReport(format, startdate, enddate) {
     .then((classes) => {
       //console.log("Classes Pagination Data: ", classes.PaginationResponse); // shows pagination data in console
       //console.log(classes);
-      let attendanceReport = [];
```

Comment:
This was the original way of returning data, before Raven's headers+data way that works better for the frontend. Sadly, this old code crashed attendance report. I think it's fixed in dev already.

Reply:
Yeah I just mentioned this on slack. The state of the attendance-endpoint.js in this branch and dev are pretty different, so this has really already been fixed.